src/codec_avm.c

```diff
@@ -625,26 +625,41 @@ static avifResult avmCodecEncodeImage(avifCodec * codec,
 
         // Set color_config() in the sequence header OBU.
         if (alpha) {
+            // AV1-AVIF specification, Section 4 "Auxiliary Image Items and Sequences":
```

Comment:
Should all of these be AVIF now for consistency?

It seems on github, the repository is av1-avif and the README has the same title. The document itself is 'AV1 Image File Format (AVIF)' so AVIF is fine for now. Hopefully future revisions for extra codecs aren't called 'AVIF'.